packages/compiler/src/compiler.ts

```diff
@@ -64,7 +65,7 @@ export class ComponentCompiler {
       (componentPath) => !this.bosSourceCache.has(componentPath)
     );
     if (unfetchedPaths.length > 0) {
-      const pathsFetch = fetchComponentSources(this.social, unfetchedPaths);
+      const pathsFetch = fetchComponentSources(this.social, unfetchedPaths, { withBlockHeight: true });
```

Comment:
Would there ever be a case where we call `fetchComponentSources()` without `{ withBlockHeight: true }`? Just curious if it's necessary to pass in the `withBlockHeight` option or if we can just always assume it's true internally within `fetchComponentSources()`.